Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.18.1 as builder
+FROM golang:1.20.3 as builder
```

```suggestion
FROM golang:1.20.3 AS builder
```
This is marked as a code-small by Sonar